### PR TITLE
react node as cell

### PR DESCRIPTION
**Pull request by kherP**

### CODE REVIEW

File: src/Table/Table.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export interface Column {
     label: string | React.ReactNode;
     accessor: string;
     canSort?: boolean;
+    cellTemplate?: (row: TableRow, cell: Cell) => React.ReactNode;
```
Comment (DanSallau) [inline src/Table/Table.tsx]: I don't see the need for this frankly speaking. Just Adding  a type reactNode to the value property should suffice. I don't think need a new property.

**Comment by DanSallau** [comment]
If casting your react value to any allows you to render you reactElement, then all i will be expecting here are the change on the typings thereby allowing reactNode . I don't see the point of adding new property here .

File: src/Table/Table.tsx (hunk 1 above)
Comment (kherP) [reply src/Table/Table.tsx]: to be honest, filtering by using another hidden columns is really just a workaround. I think we should keep this.

Comment (DanSallau) [reply src/Table/Table.tsx]: This PR address filtering or just rendering reactElement ?

File: develop/components/pages/TablePage.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import { Table, Column, TableRow, PrimaryActionButton, ActionLinkItem, TableHeader, DataItem, FilterItem, FilterProps } from "../../../src/Table";
+import { Table, Column, TableRow, Cell, PrimaryActionButton, ActionLinkItem, TableHeader, DataItem, FilterItem, FilterProps } from "../../../src/Table";
```
Comment (yousifalraheem) [inline develop/components/pages/TablePage.tsx]: You've imported `Cell` interface but never used it

File: src/Table/Table.tsx [hunk 3]
```diff
@@ -51,7 +52,7 @@ export interface ActionLinkItem {
 }
 
 export interface PrimaryActionButton {
-    label: string;
+    label: string | React.ReactNode;
```
Comment (yousifalraheem) [inline src/Table/Table.tsx]: I believe `ReactNode` interface extends `ReactText` which is a `string` or `number`
```suggestion
    label: React.ReactNode;
```

File: src/Table/Table.tsx (hunk 1 above)
Comment (yousifalraheem) [reply src/Table/Table.tsx]: I agree with @DanSallau here. It's not just here, we use the same approach will all our components. You only need a type change for `value` prop to `ReactNode` and that's should suffice

Comment (kherP) [reply src/Table/Table.tsx]: loop each `TableRow` then each `Cell` to set the value for each cell? Please reconsider.

**Pull request closed without merging**